torch/_dynamo/create_parameter_op.py

```diff
@@ -24,7 +24,7 @@ def backward(ctx, grad):
 def tracable_create_parameter(tensor, placeholder):
     with torch.set_grad_enabled(placeholder.requires_grad):
         out = TracableCreateParameter.apply(tensor, placeholder)
-    return out
+    return out.clone()
```

Comment:
Got  a scary looking error message for test `PYTORCH_TEST_WITH_DYNAMO=1 python test/test_transformers.py -k TestTransformersCPU.test_encoder_is_causal_cpu` that suggested to add `.clone()`